docs/extras/modules/data_connection/vectorstores/integrations/meilisearch.ipynb

```diff
@@ -19,23 +19,45 @@
    "source": [
     "## Setup\n",
     "\n",
-    "### Run Meilisearch\n",
+    "### Launching a Meilisearch instance\n",
+    "\n",
+    "You will need a running Meilisearch instance to use as your vector store. You can run [Meilisearch in local](https://www.meilisearch.com/docs/learn/getting_started/installation#local-installation) or create a [Meilisearch Cloud](https://cloud.meilisearch.com/) account.\n",
+    "\n",
+    "As of Meilisearch v1.3, vector storage is an experimental feature. After launching your Meilisearch instance, you need to **enable vector storage**. For self-hosted Meilisearch, read the docs on [enabling experimental features](https://www.meilisearch.com/docs/learn/experimental/vector-search). On **Meilisearch Cloud**, enable _Vector Store_ via your project _Settings_ page.\n",
+    "\n",
+    "You should now have a running Meilisearch instance with vector storage enabled. 🎉\n",
+    "\n",
+    "### Credentials\n",
+    "\n",
+    "To interact with your Meilisearch instance, the Meilisearch SDK needs a host (URL of your instance) and an API key.\n",
     "\n",
-    "TODO: Revision\n",
-    "To run this notebook you need a running Meilisearch instance with Vector Search capabilities enable.\n",
+    "**Host**\n",
     "\n",
-    "See the following documentation for how to run a Meilisearch instance:\n",
-    "https://www.meilisearch.com/docs/learn/getting_started/quick_start.\n",
+    "- In **local**, the default host is `localhost:7700`\n",
+    "- On **Meilisearch Cloud**, find the host in your project _Settings_ page\n",
     "\n",
-    "If you are looking for a hosted Meilisearch, take a look at this:\n",
-    "https://www.meilisearch.com/pricing.\n",
+    "**API keys**\n",
+    "\n",
+    "Meilisearch instance provides you with three API keys out of the box: \n",
+    "- A `MASTER KEY` — it should only be used to create your Meilisearch instance\n",
+    "- A `ADMIN KEY` — use it only server-side to update your database and its settings\n",
+    "- A `SEARCH KEY` — a key that you can safely share in front-end applications\n",
+    "\n",
+    "You can create [additional API keys](https://www.meilisearch.com/docs/learn/security/master_api_keys) as needed."
```

Comment:
Do you think it's better to explain directly here rather than just link to the documentation?

Reply:
I think it makes it easier for the reader not to have to go back and forth between docs.

Plus, the API keys thing is "stable enough" that we can be confident it won't change, so it's not like it's creating much documentation burden on our end.